test/tables.bbcode

```diff
@@ -0,0 +1,51 @@
+Simple table with caption:
+
+[table]
+[tr][th]Right[/th][th]Left[/th][th]Center[/th][th]Default[/th][/tr]
+[tr][td]12[/td][td]12[/td][td]12[/td][td]12[/td][/tr]
+[tr][td]123[/td][td]123[/td][td]123[/td][td]123[/td][/tr]
+[tr][td]1[/td][td]1[/td][td]1[/td][td]1[/td][/tr]
+[/table]
+Simple table without caption:
```

Comment:
Would it be desirable to have a blank line between the table and the following content?